Gallery/Topography/NCL_topo_1.py

```diff
@@ -0,0 +1,89 @@
+"""
+NCL_topo_1.py
+===============
+This script illustrates the following concepts:
+   - Drawing a topographic map using 1' data
+   - Drawing topographic data using GMT colormap
+   - Using wget to download a dataset
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/topo_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/topo_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import matplotlib.pyplot as plt
+import xarray as xr
+import numpy as np
+import cartopy.crs as ccrs
+from cartopy.mpl.gridliner import LongitudeFormatter, LatitudeFormatter
+import cmaps
+import wget
+
+import geocat.viz as gv
+
+###############################################################################
+# Read in data:
+
+# Use wget to retrieve and download NOAA elevation data
+url = 'https://www.ngdc.noaa.gov/mgg/global/relief/ETOPO1/data/ice_surface/grid_registered/netcdf/ETOPO1_Ice_g_gmt4.grd.gz'
+file_name = wget.download(url)
+
+# Open the dataset using xarray
+ds = xr.open_dataset(file_name)
+
+# Select the elevation data
+ds = ds.z
+
+###############################################################################
+# Plot
+
+# Generate figure and set size
+plt.figure(figsize=(12, 10))
+
+# Generate axes, using Cartopy
+projection = ccrs.PlateCarree()
+ax = plt.axes(projection=projection)
+
+# Add coastines
+ax.coastlines(zorder=10)
+
+# Plot the elevation data
+elev = ds.plot(ax=ax,
+               transform=projection,
+               cmap=cmaps.GMT_relief,
+               add_colorbar=False)
+
+# Add colorbar
+cbar = plt.colorbar(ax=ax, mappable=elev, orientation='horizontal', pad=0.1)
+cbar.ax.tick_params(
+    size=0,
+    labelsize=14)  # Remove the tick marks from the colorbar, set label size
+cbar.ax.xaxis.set_tick_params(pad=10)
+
+# Use geocat-viz utility function to format major and minor tick marks
+gv.add_major_minor_ticks(ax, labelsize=14)
+
+# Add title, remove default x and y labels
+plt.title('ETOPO1', fontweight='bold', fontsize=23, pad=15)
```

Comment:
I decided not to use the function because I don't think it allows me to make the title in bold font. I may be wrong about this though. Also, because this plot doesn't have left/right titles it didn't seem necessary to use the function because it is fairly simple to just use plt.title() for a main title.

Reply:
That's true. We still need to add that functionality to geocat.viz. This should be fine for now